test/unit-test/mqtt_agent_utest.c

```diff
@@ -258,6 +269,45 @@ MQTTStatus_t MQTT_ProcessLoop_stub( MQTTContext_t * pContext,
     return MQTTSuccess;
 }
 
+/**
+ * @brief A stub for MQTT_ProcessLoop function to be used to test multiple calls to MQTT_ProcessLoop.
+ */
+MQTTStatus_t MQTT_ProcessLoop_stub_multiple( MQTTContext_t * pContext,
+                                             uint32_t timeoutMs )
```

Comment:
This needs an additional parameter to match CMock's stub function signature
```suggestion
                                             uint32_t timeoutMs,
                                             int NumCalls )
```